requirements_all_ds.txt

```diff
@@ -31,7 +31,7 @@ phoenixdb==0.7
 # certifi is needed to support MongoDB and SSL:
 certifi>=2019.9.11
 pydgraph==2.0.2
-azure-kusto-data==0.0.35
+azure-kusto-data==2.1.1
```

Comment:
This is a significant update. Were there any breaking changes between v0 and v2.1.1?